### PR TITLE
[KR] 08.md & [KR] 08-1.md

### PR DESCRIPTION
**Pull request by marofmar**
[KR] 08.md translation and added the path for it in _config.yml.
Please review and leave your comments! Happy weekend!

### CODE REVIEW

File: docs/ko/week06/06-1.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
 ---
 lang-ref: ch.06-1
-lecturer: Yann LeCun
```
Comment (Atcold) [inline docs/ko/week06/06-1.md:3]: Why?

File: docs/ko/week06/06-1.md [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
 title: 합성곱 신경망의 응용
 authors: Shiqing Li, Chenqin Yang, Yakun Wang, Jimin Tan
-date: 2 Mar 2020
+date: 4 Mar 2020
```
Comment (Atcold) [inline docs/ko/week06/06-1.md:5]: What happened?

File: docs/ko/week06/06-1.md [hunk 3]
```diff
@@ -37,24 +34,23 @@ translator : Yujin
...
 
-합성곱 신경망은 탐지/인식<sup>detection</sup> 작업에서 높은 성과를 보이며, 이는 얼굴 인식<sup>face detection</sup> 분야에서도 예외는 아니다. 얼굴 인식을 수행하기 위해서 우리는 얼굴이 있는 이미지와 얼굴이 없는 이미지를 모아서 데이터셋을 만들고, 30 $\times$ 30 픽셀 윈도우<sup>window</sup> 등의 크기를 가진 합성곱 신경망을 학습시켜 여기에 얼굴이 있는지 없는지를 가려내게 한다. 일단 훈련이 되면, 우리는 이 모델을 새로운 이미지에 적용해보고 대략 30 $\times$ 30 픽셀 윈도우 창 내에 들어오는 얼굴이 있을시, 합성곱 신경망은 해당 위치의 출력을 밝게 표시할 것이다. 하지만 여기에는 두 개의 문제가 있다.
+합성곱 신경망은 탐지/인식<sup>detection</sup> 작업에서 높은 성과를 보이며, 이는 얼굴 인식<sup>face detection</sup> 분야에서도 예외는 아니다. 얼굴 인식을 수행하기 위해서 우리는 얼굴이 있는 이미지와 얼굴이 없는 이미지를 모아서 데이터셋을 만들고, 30 $\times$ 30 픽셀 윈도우<sup>window</sup> 등의 크기를 가진 합성곱 신경망을 학습시켜 여기에 얼굴이 있는지 없는지를 가려내게 한다. 일단 훈련이 되면, 우리는 이 모델을 새로운 이미지에 적용해보고 대략 30 $\times$ 30 픽셀 윈도우 창 내에 들어오는 얼굴이 있을시, 합성곱 신경망은 해당 위치의 출력을 밝게 표시할 것이다. 하지만 여기에는 두 개의 문제가 있다.
```
Comment (Atcold) [inline docs/ko/week06/06-1.md:45]: Please, don't add empty spaces at the end of the sentences.

File: docs/ko/week06/06-1.md [hunk 4]
```diff
@@ -37,24 +34,23 @@ translator : Yujin
...
+합성곱 신경망은 탐지/인식<sup>detection</sup> 작업에서 높은 성과를 보이며, 이는 얼굴 인식<sup>face detection</sup> 분야에서도 예외는 아니다. 얼굴 인식을 수행하기 위해서 우리는 얼굴이 있는 이미지와 얼굴이 없는 이미지를 모아서 데이터셋을 만들고, 30 $\times$ 30 픽셀 윈도우<sup>window</sup> 등의 크기를 가진 합성곱 신경망을 학습시켜 여기에 얼굴이 있는지 없는지를 가려내게 한다. 일단 훈련이 되면, 우리는 이 모델을 새로운 이미지에 적용해보고 대략 30 $\times$ 30 픽셀 윈도우 창 내에 들어오는 얼굴이 있을시, 합성곱 신경망은 해당 위치의 출력을 밝게 표시할 것이다. 하지만 여기에는 두 개의 문제가 있다. 
 
 <!-- - **False Positives**: There are many ways that a patch of an image is not a face. During the training stage, the model may not see all of them (i.e. a fully representative set.). Therefore, the model may suffer from a lot of false positives at test time. -->
 
-- **거짓 양성 오류<sup>False Positive</sup>**: 이미지의 일부가 얼굴이 아닌 경우는 매우 많다. 합성곱 신경망의 학습 과정 동안, 이 모델은 이런 이미지 전부를 보지 못할 것이다 (즉, 완전히 대표적인 데이터셋). 따라서 모델을 테스트하는 시점에서 수많은 거짓 양성 오류가 발생할 수 있다.
+- **거짓 양성 오류<sup>False Positive</sup>**: 이미지의 일부가 얼굴이 아닌 경우는 매우 많다. 합성곱 신경망의 학습 과정 동안, 이 모델은 이런 이미지 전부를 보지 못할 것이다 (즉, 완전히 대표적인 데이터셋). 따라서 모델을 테스트하는 시점에서 수많은 거짓 양성 오류가 발생할 수 있다.
```
Comment (Atcold) [inline docs/ko/week06/06-1.md:49]: Same here.

File: docs/ko/week06/06-1.md [hunk 5]
```diff
@@ -65,52 +61,52 @@ translator : Yujin
 </center> -->
 <center>
 <img src="{{site.baseurl}}/images/week06/06-1/8R3v0Dj.png" style="zoom: 30%; background-color:#DCDCDC;"/><br>
-<b>그림 2:</b> 얼굴 인식 시스템
+<b>그림 2:</b> 얼굴 인식 시스템
```
Comment (Atcold) [inline docs/ko/week06/06-1.md:64]: No additional spaces, plz.

File: docs/ko/week06/06-1.md [hunk 6]
```diff
@@ -65,52 +61,52 @@ translator : Yujin
...
 <img src="{{site.baseurl}}/images/week06/06-1/CQ8T00O.png" style="zoom: 40%; background-color:#DCDCDC;"/><br>
-<b>그림 3:</b> 다양한 스케일링 계수에 따른 얼굴 인식기 점수
+<b>그림 3:</b> 다양한 스케일링 계수에 따른 얼굴 인식기 점수 
+
 </center>
 
 
 <!-- ### Non-maximum suppression -->
-### 비최댓값 억제<sup>Non-maximum suppression</sup>
+### 비최댓값 억제<sup>Non-maximum suppression</sup>
```
Comment (Atcold) [inline docs/ko/week06/06-1.md:82]: No additional spaces.

File: docs/ko/week06/06-1.md [hunk 7]
```diff
@@ -65,52 +61,52 @@ translator : Yujin
...
 
 
 <!-- ### Non-maximum suppression -->
-### 비최댓값 억제<sup>Non-maximum suppression</sup>
+### 비최댓값 억제<sup>Non-maximum suppression</sup> 
 
 <!-- For each high-scoring region, there is probably a face underneath. If more faces are detected very close to the first, it means that only one should be considered correct and the rest are wrong. With non-maximum suppression, we take the highest-scoring of the overlapping bounding boxes and remove the others. The result will be a single bounding box at the optimum location. -->
 
-각각의 높은 점수를 보이는 영역마다 얼굴을 탐지해 냈을 것이다. 만일 첫 번째 인식된 얼굴과 굉장히 가까운 부근에서 더 많은 수의 얼굴이 인식된다면, 이는 단 한 개의 인식된 얼굴만 옳은 것, 나머지는 잘못된 것이라고 볼 수 있다. 비최댓값 억제를 이용해서, 우리는 중첩된 경계의 인식된 이미지 경계 박스<sup>bounding box</sup> 들 중에서 가장 높은 점수를 얻는 한 개의 것만 취하고, 나머지는 버린다. 이에 따라 최적의 위치에 인식된 이미지 박스 하나만 남을 것이다.
+각각의 높은 점수를 보이는 영역마다 얼굴을 탐지해 냈을 것이다. 만일 첫 번째 인식된 얼굴과 굉장히 가까운 부근에서 더 많은 수의 얼굴이 인식된다면, 이는 단 한 개의 인식된 얼굴만 옳은 것, 나머지는 잘못된 것이라고 볼 수 있다. 비최댓값 억제를 이용해서, 우리는 중첩된 경계의 인식된 이미지 경계 박스<sup>bounding box</sup> 들 중에서 가장 높은 점수를 얻는 한 개의 것만 취하고, 나머지는 버린다. 이에 따라 최적의 위치에 인식된 이미지 박스 하나만 남을 것이다.
```
Comment (Atcold) [inline docs/ko/week06/06-1.md:86]: Same.

File: docs/ko/week07/07-2.md [hunk 8]
```diff
@@ -0,0 +1,349 @@
+---
+lang-ref: ch.07-2
+title: 세부 사항 및 예제가 포함된 SSL, EBM
```
Comment (Atcold) [inline docs/ko/week07/07-2.md:3]: `lecturer:` is missing.

**Comment by marofmar** [comment]
fixed 06-1, 07-2, and also edited spaces for 08 too!
Sorry for the confusion and mistakes.

**Comment by marofmar** [comment]
[KR] 08-1.md

**Comment by Atcold** [comment]
I'm not sure about this:

image

And there are way too many conflicts on `07-2.md`. Please, rebase your branch on top of `master`.

File: docs/ko/week08/08.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+이 섹션에서는 정규화된 잠재변수 EBM의 조건부<sup>conditional</sup> 그리고 무조건부<sup>unconditional</sup> 모델 개념을 포함해 자세히 논의한다. 다음으로, ISTA, FISTA 그리고 LISTA 알고리즘에 대해 이야기하고, 희소 코딩<sup>sparse coding</sup> 예제와 합성곱 희소 인코더<sup>convolutional sparse encoders</sup>로부터 학습된 필터에 대해 알아본다. 마지막으로 Variational 오토인코더<sup>Variational Auto-Encoders</sup>와 이것과 관련된 기본 개념에 대해 이야기한다.  
+
+<!-- ## Practicum -->
+## 실습
+<!-- In this section, we discussed a specific type of generative model called Variational Autoencoders and compared their functionalities and advantages over Classic Autoencoders. We explored the objective function of VAE in detail, understanding how it enforced some structure in the latent space. Finally, we implemented and trained a VAE on the MNIST dataset and used it to generate new samples. -->
+이 섹션에서는 Variational 오토인코더<sup>autoencoders</sup>라는 특정한 생성 모델<sup>generative models</sup>에 대해 논의하고 고전적인<sup>Classic</sup> 오토인코더와 기능 및 장점에 대해 비교한다. 우리는 잠재 공간<sup>latent space</sup>에서 VAE가 어떤 구조를 어떻게 적용하는지 이해하며 VAE의 목적 함수를 자세히 알아본다. 마지막으로 MNIST 데이터셋에 대해 VAE를 구현하고 훈련시켜서 이것이 새로운 샘플을 생성하도록 한다.
```
Comment (skyil7) [inline docs/ko/week08/08.md:25]: Variational Autoencoder는 변이형 오토인코더라는 명칭이 국내에서 많이 사용되고 있어, `변이형 오토인코더<sup>Variational Autoencoder</sup>`과 같이 적어주시면 좋을 것 같습니다.

번역 수고하셨습니다!!

Comment (marofmar) [reply docs/ko/week08/08.md]: 수정했습니다!

**Comment by Atcold** [comment]
Great, thanks!

**Pull request merged**